users/permissions.py

```diff
@@ -11,13 +11,16 @@ def has_object_permission(self, request, view, obj):
 class IsAdminUser(permissions.BasePermission):
```

Comment:
I think better name for this class would be `AuthenticatedAdmin`. Classes in general should have more general names and than function inside should be more precise. If you have maybe some ideas about names fell free to write :)

Reply:
Done